content/docs/command-reference/remote/add.md

```diff
@@ -334,6 +334,33 @@ $ dvc remote add -d myremote hdfs://user@example.com/path/to/dir
 
 <details>
 
+### Click for WebHDFS
+
+**HDFS and WebHDFS:**
```

Comment:
Looks great @tomasfarias ! One last thing from me - let's add into `dvc remote add` HDFS, `dvc remote modify` HDFS/WebHDFS sections a link/note/something that mentions about two options and that details could be found here.

Reply:
Added a note in `dvc remote add` HDFS and two links + note in `dvc remote modify` HDFS/WebHDFS as you suggested 👍 . Let me know if they are clear enough.